packages/supabase/lib/src/supabase_stream_builder.dart

```diff
@@ -141,71 +139,71 @@ class SupabaseStreamBuilder extends Stream<SupabaseStreamEvent> {
   Future<void> _getStreamData() async {
     final currentStreamFilter = _streamFilter;
     _streamData = [];
-    String? realtimeFilter;
+    PostgresChangeFilter? realtimeFilter;
     if (currentStreamFilter != null) {
-      if (currentStreamFilter.type == _FilterType.inFilter) {
+      if (currentStreamFilter.type == PostgresChangeFilterType.inFilter) {
```

Comment:
I think this if can be removed as both cases are now the same.

Reply:
I might not be following you, but wouldn't we still need to check if the filter being applied is `inFilter` here?

Reply:
I don't think so, because the same `PostgresChangeFilter` is created in both cases. That class now handles the special formatting for the `inFilter`.

Reply:
Oh yeah, now I see it. Great catch! 

Will edit this one.